src/App.js

```diff
@@ -2,20 +2,27 @@ import React from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
-
+function useSemiPersistentState () {
+  const [todoList, setTodoList] = React.useState(JSON.parse(localStorage.getItem('savedTodoList')));
+    React.useEffect(() => {
```

Comment:
you can import useEffect and useState from the react library in the import statement with importing React. so your import statement will end up looking like: 

import React, { useState, useEffect } from 'react';

will save you a bit of typing